code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -144,11 +153,13 @@
 	icon_state = "poison_headcrab"
 	icon_living = "poison_headcrab"
 	icon_dead = "poison_headcrab_dead"
-	health = 60
-	maxHealth = 60
+	health = 80
```

Comment:
Doesn't dodging give >50% chance of immunity to all ranged attacks?
If so, that already pretty much negates half of the weapons you can use against these.
Further buffing their health so they take 8 harmbatons per headcrab to kill means that clearing out a nest of these will take 8*num_headcrabs hits, which could easily be over 100 attacks. Not including any necessary to destroy the nest.

Maybe giving them dodging is a bad idea. Raising their health disproportionately makes them more difficult for assistants, but sec doesn't care as much as it will just use 20 damage/shot lasers.

Giving them dodging massively disadvantages sec trying to fight them, as sec is the only type of crew likely to have ranged weapons, and sec are MEANT to be able to take these out much easier than regular crew.

Reply:
From what I check, they don't give immunity to ranged attack but rather make them engage in erratic movement.

Health wise seems to be fine